src/solver/optim-model-filler/include/antares/solver/optim-model-filler/LinearExpression.h

```diff
@@ -18,162 +18,82 @@
  * You should have received a copy of the Mozilla Public Licence 2.0
  * along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
  */
-
 #pragma once
```

Comment:
This file should be split in 2 parts.
One for class **LinearExpression** and one for **TimeDependentLinearExpression**